packages/storybook/stories/wc-helpers.jsx

```diff
@@ -10,13 +10,13 @@ import {
   PRIMARY_STORY,
 } from '@storybook/addon-docs/blocks';
 
-import componentDocs from 'web-components/component-docs.json';
+import {webComponentDocs} from '@department-of-veterans-affairs/component-library';
```

Comment:
The `core` webpack config bundles the JSON in with the exports, rather than leaving it as its own file.